docs/user/user_doc.md

```diff
@@ -124,7 +124,7 @@ $ aixplain get image-repo-login [--api-key <TEAM_API_KEY>]
 
```

Comment:
You may want to mention here that the image needs to be built following the standard bellow:

```
docker build . -t 535945872701.dkr.ecr.us-east-1.amazonaws.com/<repositoryName>:<tag>
```

Reply:
Fixed!